design/20240122.scarf.md

```diff
@@ -59,4 +59,17 @@ Going forward, we would encourage users to use the new download paths by specify
 Any users downloading from secure environments with limited internet connections through firewall restrictions will need to add "allowed" rules for the Scarf gateway domain in addition to any existing rules for the image repository, such as quay.io. These should be clearly documented.
 
 ### Known issues/limitations
+
 - Currently, the Scarf service only allows for custom domains and doesn't include custom paths. When speaking with members of the Scarf organisation, this is due to a technical limitation as the path is used in the image identification/verification process. Scarf is investigating a workaround; however, we may need to consider an additional hosting location/service to allow us to remove "jetstack" from the download path. An additional hosting location will increase existing maintenance and deployment process overheads.
+
+## Decision to Reassess Scarf.sh Integration Proposal
+
+Following a comprehensive review of the proposed integration of Scarf.sh into cert-manager, we have decided to discontinue this initiative at this time. While Scarf.sh provides innovative tools for enhancing software distribution insights, our assessment indicates that the additional data it offers does not sufficiently surpass the insights we already gather through our existing hosting infrastructure.
```

Comment:
It would be great to give a specific date, like "As of November 2024, ..."

Reply:
Please add more context around what we have tried, how long we have tried it, the results we got, and what we haven't tried and why (e.g., we haven't tried replacing the Helm chart's default image to use Scarf).

Also, I'd suggest keeping a neutral tone. Thse new parapgraphs feel overly verbose, and no longer feels like a design documents (and more of an letter of apology). Let's get to the point (no need for "innovative" etc). This document is about facts, I don't think we need the diplomatic phrasing. Scarf's team will understand and will be grateful for the explanations around what we tried and didn't try.